File: library/src/main/java/moe/leekcake/live2dforjvm/framework/type/id/CubismId.java
package moe.leekcake.live2dforjvm.framework.type.id;

import moe.leekcake.live2dforjvm.framework.jni.id.CubismIdJNI;
import moe.leekcake.live2dforjvm.core.type.AutoPointer;

public class CubismId extends AutoPointer {
    public CubismId(long pointer) {
        super(pointer);
    }

    public String getString() {
        return CubismIdJNI.GetString(pointer);
    }
}

File: library/src/main/kotlin/moe/leekcake/live2dforjvm/framework/type/id/CubismId.kt
package moe.leekcake.live2dforjvm.framework.type.id

import moe.leekcake.live2dforjvm.framework.id.CubismIdJNI

class CubismId(val pointer: Long) {
    fun getString(): String {
        return CubismIdJNI.CubismId_GetString(pointer);
    }
}